package.json

```diff
@@ -38,15 +38,15 @@
   "homepage": "https://github.com/bbc/psammead#readme",
   "dependencies": {
     "styled-components": "^4.4.1",
-    "styled-normalize": "^8.0.4"
+    "styled-normalize": "^8.0.7"
```

Comment:
I notice in the description and changelog, this is marked as a developer dependency bump, but `styled-normalize` is used within our application. Was it meant to be included?